monitors/CRMonitor.js

```diff
@@ -101,40 +112,47 @@ class CRMonitor {
         });
       }
     }
-  };
+  }
 
-  _getPositionInformation = address => {
-    const positionInfo = this.empClient.getAllPositions().filter(position => position.sponsor == address);
-    if (positionInfo.length == 0) {
-      return null;
-      // there should only ever be one position information object per address
-    } else return positionInfo[0];
-  };
+  _getPositionInformation(address) {
+    return this.empClient.getAllPositions().find(position => position.sponsor == address);
```

Comment:
I assume this returns `null` if it can't find the element?

Reply:
It returns `undefined`, but that == `null` and we use == rather than === throughout this file.